modules/KalturaSupport/resources/mw.KWidgetSupport.js

```diff
@@ -302,10 +301,12 @@ mw.KWidgetSupport.prototype = {
 
 					if (action.pattern && action.replacement) {
 						var regExp=new RegExp(action.pattern, "i");
-						if (!this.originalServiceUrl){
-							this.originalServiceUrl = mw.config.get('Kaltura.playManifestServiceUrl');
+						var originalServiceUrl = window.parent.mw.getConfig('Kaltura.originalServiceUrl');
+						if (!originalServiceUrl){
+							originalServiceUrl = mw.config.get('Kaltura.playManifestServiceUrl');
+							window.parent.mw.setConfig('Kaltura.originalServiceUrl', originalServiceUrl)
```

Comment:
also, is it possible that `originalServiceUrl` is `null` or `undefined`?

Reply:
yes